transcom/mymove
# [MB-14973] Fix the current days in SIT number and the total days remaining

## [Jira ticket](https://dp3.atlassian.net/browse/MB-14973)

## Summary

* Fix total days remaining
* update test
* Fix current days in SIT


## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

See Checkboxes in the test steps below.

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/run-storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/about/application-setup/milmove-local-client/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/about/development/testing)

### How to test

1. Access the Office App
2. Login as a TOO
3. Navigate to `/moves/PENDNG/mto/`
     - [x] Current days in sit should be equal to Total days used - Previous days in SIT
5. Click review request
     - [ ] Current days in sit should be equal to Total days used - Previous days in SIT
     - [x] Total remaining days should be Total approved days - Total days used - 1
7. Navigate to `/moves/SITEXT/mto`
     - [x] Current days in sit should be equal to Total days used - Previous days in SIT

### Frontend


## Screenshots
